src/app/customer/getCreateCustomerValidationSchema.ts

```diff
@@ -62,6 +63,7 @@ export default memoize(function getCreateCustomerValidationSchema({
                 .matches(alpha, description || language.translate('customer.password_letter_required_error'))
                 .min(minLength, description || language.translate('customer.password_under_minimum_length_error'))
                 .max(100, language.translate('customer.password_over_maximum_length_error')),
+            requiresMarketingConsent: boolean().required(),
```

Comment:
Not sure what you're trying to do here but `requiresMarketingConsent` is not the input that needs validating.